src/Yoti.Auth/Image.cs

```diff
@@ -1,32 +1,49 @@
-﻿using static Yoti.Auth.YotiAttributeValue;
-
-namespace Yoti.Auth
+﻿namespace Yoti.Auth
 {
     public class Image
     {
-        public TypeEnum Type { get; set; }
+        public Image(AttrpubapiV1.ContentType type, byte[] content)
+        {
+            Content = content;
+            Type = type;
+        }
 
-        public byte[] Data { get; set; }
+        public AttrpubapiV1.ContentType Type { get; private set; }
 
-        public string Base64URI
+        public byte[] Content { get; private set; }
+
+        public string MIMEType
         {
             get
             {
                 switch (Type)
                 {
-                    case TypeEnum.Jpeg:
-                        return "data:image/jpeg;base64," + Conversion.BytesToBase64(Data);
-                    case TypeEnum.Png:
-                        return "data:image/png;base64," + Conversion.BytesToBase64(Data);
+                    case AttrpubapiV1.ContentType.Jpeg:
```

Comment:
If we take the approach of using different subclasses for different image types, then:
1. This _Image_ class does not need to know about _AttrpubapiV1.ContentType_ i.e. it becomes agnostic towards our underlying implementation - it knows nothing about Protobuf (and it shouldn't really)
2. Specific classes will know how to represent themselves - we will not need multiple switch-case statements